docs/section11/Rule_11-7.md

```diff
@@ -0,0 +1,47 @@
+# Service_Water_Heating - Rule 11-7  
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-7  
+
+**Rule Description:** Except in buildings that will have no service water heating loads,  the service water heating systems type in the baseline building design shall be as specified in Table G3.1.1-2 for each building area type in the proposed design. 
```

Comment:
At one point I remember discussion about the specific building types that might not have service water heating loads like parking garage and maybe a few others. Do we want to consider that as an approach for determining applicability?

Reply:
updated to check for parking